src/f77_interface.F

```diff
@@ -18,126 +18,130 @@
-                                              mp_comm_world,&
-                                              mp_para_env_release,&
-                                              mp_para_env_type,&
-                                              mp_world_finalize,&
-                                              mp_world_init
-   USE metadynamics_types,              ONLY: meta_env_type
-   USE metadynamics_utils,              ONLY: metadyn_read
-   USE mixed_environment_types,         ONLY: mixed_environment_type
-   USE mixed_main,                      ONLY: mixed_create_force_env
-   USE mp_perf_env,                     ONLY: add_mp_perf_env,&
-                                              get_mp_perf_env,&
-                                              mp_perf_env_release,&
-                                              mp_perf_env_retain,&
-                                              mp_perf_env_type,&
-                                              rm_mp_perf_env
-   USE nnp_environment,                 ONLY: nnp_init
-   USE nnp_environment_types,           ONLY: nnp_type
-   USE offload_api,                     ONLY: offload_get_chosen_device,&
-                                              offload_get_device_count,&
-                                              offload_set_chosen_device
-   USE periodic_table,                  ONLY: init_periodic_table
-   USE pw_gpu,                          ONLY: pw_gpu_finalize,&
-                                              pw_gpu_init
-   USE pwdft_environment,               ONLY: pwdft_init
-   USE pwdft_environment_types,         ONLY: pwdft_env_create,&
-                                              pwdft_environment_type
-   USE qmmm_create,                     ONLY: qmmm_env_create
-   USE qmmm_types,                      ONLY: qmmm_env_type
-   USE qmmmx_create,                    ONLY: qmmmx_env_create
-   USE qmmmx_types,                     ONLY: qmmmx_env_type
-   USE qs_environment,                  ONLY: qs_init
-   USE qs_environment_types,            ONLY: get_qs_env,&
-                                              qs_env_create,&
-                                              qs_environment_type
-   USE reference_manager,               ONLY: remove_all_references
-   USE string_table,                    ONLY: string_table_allocate,&
-                                              string_table_deallocate
-   USE timings,                         ONLY: add_timer_env,&
-                                              get_timer_env,&
-                                              rm_timer_env,&
-                                              timer_env_release,&
-                                              timer_env_retain,&
-                                              timings_register_hooks
-   USE timings_types,                   ONLY: timer_env_type
-   USE virial_types,                    ONLY: virial_type
+   USE base_hooks, ONLY: cp_abort_hook, &
```

Comment:
Is the indentation change here intentional?

Reply:
Ooof... Looks like another issue with the `make pretty` command when using `#ifdef`/`#if defined`.

Reply:
Won't fix. Let's deal with this on CP2K's side.